check/format-incremental

```diff
@@ -112,16 +112,8 @@ if (( only_print == 1 )); then
     args+=("--check" "--diff")
 fi
 
-# Warn users about bug in black: https://github.com/psf/black/issues/1629
-# Once that is fixed upstream, we can just do:
-# black "${args[@]}" "${format_files[@]}"
-# exit $?
-LOGS="$(black "${args[@]}" "${format_files[@]}" 2>&1)"
+black "${args[@]}" "${format_files[@]}"
```

Comment:
Do we need to re-run the formatter on all files?

Reply:
> Do we need to re-run the formatter on all files?

No, the previous version would still exit with error if black failed.
The removed code was only showing a message about the bug in black.